mitodl/edx-sga
# Add download all submissions

#### What are the relevant tickets?
fixes https://github.com/mitodl/edx-sga/issues/184

#### What's this PR do?
- It collect submissions of all students in background and make a zip file for download

####  How to test?
- Add sga to you course from studio
- Use different edX accounts to upload assignments, i recommend 9mb each assignment.
- click `collect all submissions`
  - <img width="401" alt="screen shot 2017-11-30 at 9 04 02 pm" src="https://user-images.githubusercontent.com/10431250/33440555-119ecf54-d612-11e7-902c-64c88db3f3fd.png">
- sit back, when zip file will be created you will see the below link.
- The after a while click down `Download ALL Submissions` links  
  -  <img width="394" alt="screen shot 2017-11-30 at 9 05 22 pm" src="https://user-images.githubusercontent.com/10431250/33440643-54a8f3d8-d612-11e7-8f5e-0a7fb1623809.png">

 
@pdpinch 

<img width="1158" alt="screen shot 2017-12-02 at 1 59 48 am" src="https://user-images.githubusercontent.com/10431250/33503654-6146b658-d706-11e7-97a4-113886c44d3c.png">

<img width="1220" alt="screen shot 2017-12-02 at 2 53 20 am" src="https://user-images.githubusercontent.com/10431250/33505130-05764810-d70c-11e7-87da-17395206b4ff.png">


<img width="1221" alt="screen shot 2017-12-02 at 2 00 09 am" src="https://user-images.githubusercontent.com/10431250/33503655-61d2030c-d706-11e7-9799-407aea32ec2a.png">
